src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -1582,7 +1582,12 @@
                 scope: $scope,
                 backdrop: "static",
                 keyboard: false,
-                ariaLabelledBy: "sync-dest-modal"
+                ariaLabelledBy: "sync-dest-modal",
+                controller: "SyncDestModalController",
+                resolve: {
+                    isSynced: isSyncDestOn,
+                    syncDestDescription: () => $scope.selectedSyncDest.description,
+                }
```

Comment:
I think this $scope.displaySyncDestModal() function belongs inside the SyncDestModalController.

Reply:
Sorry, you can ignore this comment.